torch/nn/modules/channelshuffle.py

```diff
@@ -46,6 +46,8 @@ def __init__(self, groups: int) -> None:
         self.groups = groups
 
     def forward(self, input: Tensor) -> Tensor:
+        if input.numel() == 0:
+            return input
```

Comment:
I think this check can be removed since `F.channel_shuffle()` already handles the empty input case